pkg/build/builder/sti.go

```diff
@@ -384,3 +393,13 @@ func scriptProxyConfig(build *api.Build) (*s2iapi.ProxyConfig, error) {
 	}
 	return config, nil
 }
+
+func copyArtifactSourceList(artifactsMapping []api.ImageSourcePath) (volumeList []s2iapi.VolumeSpec) {
```

Comment:
Maybe name it like `asVolumeList()`? Or `toVolumeList()`? Or also use verb like `transform`? Because for me we're not copying here, but creating a new object.